models/stics/R/write.config.STICS.R

```diff
@@ -1752,5 +922,193 @@ write.config.STICS <- function(defaults, trait.values, settings, run.id) {
   writeLines(jobsh, con = file.path(settings$rundir, run.id, "job.sh"))
   Sys.chmod(file.path(settings$rundir, run.id, "job.sh"))
   
-
+  
 } # write.config.STICS
+
+
+# ==================================================================================================#
+#' Function to translate pecan param names and units to stics names and units.
+#' @export
+#' @param trait.values trait.values, list
+#' @return translated list
+#' @author Quentin Bell
+# Based on pecan2lpjguess function by Istem Fer https://github.com/PecanProject/pecan/blob/develop/models/lpjguess/R/write.config.LPJGUESS.R#L229
+pecan2stics <- function(trait.values){
+  
+  # TODO :match all stics and pecan names
+  vartable <- tibble::tribble(
```

Comment:
you can avoid needing to declare a new import by using the version in dplyr:
```suggestion
  vartable <- dplyr::tribble(
```